pkg/operation/seed/seed.go

```diff
@@ -422,6 +422,17 @@ func RunReconcileSeedFlow(
 			}
 		}
 
+		//TODO: Remove after next release
+		flbpc := &schedulingv1.PriorityClass{ObjectMeta: metav1.ObjectMeta{Name: "fluent-bit"}}
+		if err := k8sSeedClient.Client().Get(ctx, kutil.Key("fluent-bit"), flbpc); client.IgnoreNotFound(err) != nil {
+			return err
+		}
+		if (err == nil || client.IgnoreNotFound(err) == nil) && flbpc.Value < 101 {
+			if err := k8sSeedClient.Client().Delete(ctx, flbpc); client.IgnoreNotFound(err) != nil {
+				return err
+			}
+		}
```

Comment:
Just choose the right value
```suggestion
		if err := deletePriorityClassIfValueNotTheSame(ctx, k8sSeedClient.Client(), "fluent-bit", 120); err != nil {
			return err
		}
```

Reply:
thanks